src/client.js

```diff
@@ -1,24 +1,53 @@
 /* eslint-env browser */
 import React from 'react';
+
+// Provider makes the store available to all connect() calls in the
+// component hierarchy below. (connect() is what connects a React
```

Comment:
What do you think about adding something like `gives smart components access to the store and allows them to call dispatch`?